agent/functional_tests/tests/generated/simpletests_windows/simpletests_generated_windows_test.go

```diff
@@ -147,6 +147,46 @@ func TestSimpleExit(t *testing.T) {
 
 }
 
+// TestTaskLocalVolume Verify that task specific Docker volume works as expected
+func TestTaskLocalVolume(t *testing.T) {
+
+	// Parallel is opt in because resource constraints could cause test failures
+	// on smaller instances
+	if os.Getenv("ECS_FUNCTIONAL_PARALLEL") != "" {
+		t.Parallel()
+	}
+	agent := RunAgent(t, nil)
+	defer agent.Cleanup()
+	agent.RequireVersion(">=1.19.0")
```

Comment:
lgtm, only concern is this might have to change since private registry will bump to 1.19.0 and this might need 1.20.0? 
@richardpen ?

Reply:
this is sitting on a separate branch, so if we need to push your work out separately on 19, then we'll update this to 20 when we merge to dev.